lang/factory.go

```diff
@@ -17,6 +18,19 @@ var languages = map[string]func() (core.Language, error){
 	},
 }
 
+func AllLanguages() []core.Language {
```

Comment:
Should not silently fail especially for libraries / framework stuff. It should:

```suggestion
func AllLanguages() ([]core.Language, error) {
```

So that the decision of handling or failing should be on the client code

Reply:
ack